### PR TITLE
Update font-kit.

### PR DESCRIPTION
**Pull request by jdm**
This shouldn't be merged until there's a new font-kit release, but I wanted to be sure that I don't lose these change.

### CODE REVIEW

File: src/draw_target.rs [hunk 1]
```diff
@@ -771,13 +773,15 @@ impl DrawTarget {
             let bounds = font.raster_bounds(
                 *id,
                 point_size,
-                &fk::FontTransform::new(self.transform.m11, self.transform.m21, self.transform.m12, self.transform.m22),
-                &(self.transform.transform_point(*position)),
+                Transform2F::row_major(self.transform.m11, self.transform.m12, self.transform.m21, self.transform.m22, 0., 0.).translate(Vector2F::new(position.x, position.y)),
```
Comment (jdm) [inline src/draw_target.rs:776]: I honestly have no idea if there is the correct way to write this.

File: src/draw_target.rs [hunk 2]
```diff
@@ -798,8 +802,7 @@ impl DrawTarget {
                 &mut canvas,
                 *id,
                 point_size,
-                &fk::FontTransform::new(self.transform.m11, self.transform.m21, self.transform.m12, self.transform.m22),
-                &position,
+                Transform2F::row_major(self.transform.m11, self.transform.m12, self.transform.m21, self.transform.m22, 0., 0.).translate(Vector2F::new(position.x, position.y)),
```
Comment (jdm) [inline src/draw_target.rs:805]: Same here.

File: src/draw_target.rs (hunk 1 above)
Comment (jrmuizel) [reply src/draw_target.rs:776]: Does @pcwalton know?

Comment (pcwalton) [reply src/draw_target.rs:776]: Looks right to me.

**Comment by atouchet** [comment]
font-kit 0.6 is now out.

**Comment by jdm** [comment]
This is now in a state where it can be merged.

**Comment by atouchet** [comment]
Can this be updated to font-kit 0.7?

**Comment by jdm** [comment]
Done.

**Pull request merged**